src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -426,8 +551,10 @@ public String executeHttpGet(String urlString) throws IOException {
 			PrintWriter out = resp.getWriter();
 			out.print(o.toString());
 			out.flush();
+		} else if (uri.equals(SUBSCRIBE_URI)) {
+			handleSubscribe(req, resp);
```

Comment:
Excellent question:  Our redirector returns the status 302 Found.  I am not certain how the HTTP client used by AWS will respond to a 302. It might issue a POST to the redirected URL or it might issue a GET.  (See https://en.wikipedia.org/wiki/HTTP_302). This bit of code makes our system robust in the face of unexpected behavior by AWS.